python/delta_sharing/rest_client.py

```diff
@@ -203,6 +208,24 @@ def __auth_basic(self, profile):
             }
         )
 
+    def set_delta_format_header(self):
```

Comment:
we should need update on this function later. because set_delta_format_header shouldn't set both delta/parquet in the header.